runtime/gc_glue_java/ObjectModel.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2019 IBM Corp. and others
+ * Copyright (c) 1991, 2020 IBM Corp. and others
```

Comment:
This file has not been changed after all, so copyrights should not be updated and file should be excluded from change set